pom.xml

```diff
@@ -181,6 +181,11 @@
         <activeMQVersion>5.8.0</activeMQVersion>
         <commonsLangVersion>2.6</commonsLangVersion>
 
+        <!-- Elasticsearch -->
+        <elasticsearchMavenPluginVersion>2.0</elasticsearchMavenPluginVersion>
+        <!-- Keep in sync with the dependency required by Elasticsearch -->
+        <elasticsearchGroovyVersion>2.4.4</elasticsearchGroovyVersion>
```

Comment:
yes we can move them to the root pom, we have no strict rules about that, more like a pragmatic approach:

My general habit is to have them in the root pom when the property (the version) needs to be shared across multiple modules.
When it comes to integration testing details, sometimes we intentionally needed (using the past as I think all such cases are gone nowadays) different versions in different modules.

In this case since I tend to believe that the version doesn't "belong" in the parent pom as it's just a testing detail (we're attempting to not couple support to a specific ES version).. although we probably want to test with the same version when it will come to integration tests within Wildfly so then it would be more practical to have these defined in the parent pom.